### PR TITLE
ADAPT-4222: Updating redirects with new locations

### PR DESCRIPTION
**Pull request by buttonwillowsix**
# READY FOR REVIEW

# Summary
- SAA has moved Trustees, Volunteer leadership, and Beyond the Farm onto homesite. These redirects represent changes to existing redirects for the first two sites, and new redirecgs for BTF.

# Review By (Date)
- No set date. Soon would be nice!

# Criticality
- 5

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Go to a URL like <localhost>/get/page/volunteering/beyondthefarm/
3. Verify that it goes to <localhost>/volunteer/beyond-the-farm/

## Front End Validation
- [ ] Is the markup using the appropriate semantic tags and passes HTML validation?
- [ ] Cross-browser testing has been performed?
- [ ] Automated accessibility scans performed?
- [ ] Manual accessibility tests performed?
- [ ] Design is approved by @ user?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADAPT-4222
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Review by mattanglin — APPROVED** [review]
lgtm! 👍🏼 
The redirects are working as expected! I assume that the target pages will eventually exist within the Alumni Homesite? Currently the newly added targets (like `/volunteer/beyond-the-farm/`) just get redirected to the cardinalalumni.stanford.edu 404 page (with the correctly, newly added 301).

File: static/_redirects [hunk 1]
```diff
@@ -327,6 +327,10 @@
 /get/page/perks/alumniemail/faqtroubleshooting /help/alumni-email-faqs/ 301!
 /get/page/life-long-learning/learn /resources/learn/ 301!
 /get/page/life-long-learning/learn-join /resources/newsletters/ 301!
+/get/page/volunteering/beyondthefarm/ /volunteer/beyond-the-farm/ 301!
```
Comment (mattanglin) [inline static/_redirects:330]: Just as an FYI Netlify normalizes trailing slashes on the source path so including/omitting it has the same redirect result

Comment (buttonwillowsix) [reply static/_redirects:330]: Good to know!

Comment (buttonwillowsix) [reply static/_redirects:330]: Hm, /volunteer/beyond-the-farm/ should go to https://alumni.stanford.edu/volunteer/beyond-the-farm/

Comment (mattanglin) [reply static/_redirects:330]: > Hm, /volunteer/beyond-the-farm/ should go to https://alumni.stanford.edu/volunteer/beyond-the-farm/

Yes! Sorry, I should've mentioned that I checked it in the Netlify deploy preview (which builds from the Storyblok DEV instance). I verified the redirects, but I'm guessing those pages don't exist in dev. We should be all good! 👍🏼

**Comment by sherakama** [comment]
I'm going to go ahead and merge this for @buttonwillowsix

**Pull request merged**